eng/targets/Imports.targets

```diff
@@ -364,5 +364,12 @@
            Condition="'@(ReferencesCopiedInThisBuild)' != ''"/>
   </Target>
 
+  <!-- Workaround for https://github.com/dotnet/runtime/pull/84936 -->
+  <Target Name="ExcludeSlowJSImportGenerator" AfterTargets="ResolveTargetingPackAssets">
+    <ItemGroup>
+      <Analyzer Remove="@(Analyzer)" Condition="'%(FileName)' == 'Microsoft.Interop.JavaScript.JSImportGenerator' AND '%(Analyzer.NuGetPackageId)' == 'Microsoft.NETCore.App.Ref'" />
```

Comment:
Why do we need the second half of this condition here? Would assume the first is sufficient.